### PR TITLE
Vintage archive

### PR DESCRIPTION
**Pull request by pipo02mix**
### What this PR does / why we need it

Towards https://github.com/giantswarm/giantswarm/issues/29615
RFC https://github.com/giantswarm/rfc/pull/84/files

Here I am moving all content, but "changes", to the `vintage` subpath. At the same time I added aliases to all vintage entries to be resolved by old URL

### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

**Review by LolloneS — APPROVED** [review]

File: src/content/vintage/advanced/cluster-management/custom-admission-controllers/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ user_questions:
   - How can I create and deploy my own admission controller?
 last_review_date: 2023-02-07
```
Comment (pipo02mix) [inline src/content/vintage/advanced/cluster-management/custom-admission-controllers/index.md:11]:
```suggestion
last_review_date: 2024-02-16
```

File: scripts/update-helm-chart-reference/main.sh [hunk 2]
```diff
@@ -15,11 +15,14 @@ docker run --rm \
 
 for file in ${DESTINATION}/*.md; do
   if grep -q '\--&gt;' $file; then
-    sed -i.bak 's/\&lt;.*\&gt;//g' $file
+    # Remove HTML comment lines
+    perl -i -pe "BEGIN{undef $/;} s/\&lt;.*\&gt;//gsm" $file
+    # Remove empty lines
+    awk 'NF > 0 || NR == 1 {blank=0} NF == 0 {blank++} blank < 2 {print}' $file > $file
+    # Remove trailing whitespace
+    sed -i 's/[ \t]*$//' $file
```
Comment (marians) [inline scripts/update-helm-chart-reference/main.sh:23]: What is this change about?

**Review by marians — APPROVED** [review]
Great! The next important thing we would need is a generic disclaimer box regarding what vintage actually means.

File: scripts/update-helm-chart-reference/main.sh (hunk 2 above)
Comment (pipo02mix) [reply scripts/update-helm-chart-reference/main.sh:23]: ups that one was unintentionally copied from helm chart configuration work I am doing in parallel

**Comment by pipo02mix** [comment]
Disclaimer added here -> https://github.com/giantswarm/docs/pull/2126

File: scripts/validate-front-matter/script.py [hunk 3]
```diff
@@ -14,8 +14,9 @@
 
 # Some path config
 path         = 'src/content'
+vintage_path = 'src/content/vintage'
```
Comment (pipo02mix) [inline scripts/validate-front-matter/script.py:17]: @marians I have included this

**Comment by marians** [comment]
Have you tried whether the Vintage disclaimer and the cookie banner overlap? What takes precedence? The vintage disclaimer should not hide the cookie consent banner.

**Comment by pipo02mix** [comment]
> Have you tried whether the Vintage disclaimer and the cookie banner overlap? What takes precedence? The vintage disclaimer should not hide the cookie consent banner.

cookie has higher z index, so it has preference

**Pull request merged**